integrations/Pendo/browser.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-underscore-dangle */
+import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+import { flattenJsonPayload } from "../../utils/utils";
+
+class Pendo {
+  constructor(config, analytics) {
+    this.analytics = analytics;
+    this.apiKey = !config.apiKey ? "" : config.apiKey;
+    this.isDebugMode = config.enableDebugMode;
+    this.name = "PENDO";
+    this.setIntervalHandler = undefined;
+    logger.debug("Config ", config);
+  }
+
+  init() {
+    window.pendo = {};
+    window.pendoCli = undefined;
+    window.pendo.apiKey = this.apiKey;
+    ScriptLoader(
+      "pendo-fs",
+      `https://cdn.pendo.io/agent/static/${this.apiKey}/pendo.js`
+    );
+    logger.debug("===in init Pendo===");
+
+    // Make a call to initPendo until pendoCli gets references to pendo object.
+    this.setIntervalHandler = setInterval(this.initPendo.bind(this), 1000);
+  }
+
+  initPendo() {
+    if (this.isReady()) {
+      window.pendoCli = window.pendo;
```

Comment:
check earlier comment to decide if this is necessary

Reply:
This line of code is not required anymore.